### PR TITLE
insert query parts within configuration example and load the full extracted graph

### PR DESCRIPTION
**Pull request by burnout87**
the full graph (`ttl` format) will be written over the output html file

### CODE REVIEW

**Comment by burnout87** [comment]
addresses also [](https://github.com/oda-hub/renku-aqs-graph-library/issues/8 ), as part of the query is passed via the configuration

**Comment by burnout87** [comment]
also migrated to renku-python `2.1.0`

File: README.md [hunk 1]
```diff
@@ -123,3 +123,20 @@ two json files have been loaded, and the related checkbox will enable/disable th
 
 The functionalities for the graph drawing and its behavior are developed in javascript and are avaialble
 at the following [repository](https://github.com/oda-hub/renku-aqs-graph-library/).
+
+
+# Installation of the plugin
+
+Currently, the plugin is developed using the version `2.1.0` of [renku-python](https://github.com/SwissDataScienceCenter/renku-python). Please make sure such version is installed by running:
```
Comment (volodymyrss) [inline README.md:130]: it's good to have the note of course, but could you also make a check for renku version while importing the plugin? With an instructive error message

Comment (burnout87) [reply README.md:130]: how could I access to the renku version specified within `setup.py`? So to compare it with the installed one...

Comment (volodymyrss) [reply README.md:130]: Usually something like this works:

```python
from packaging import version
import renku

version.parse(renku.__version__) > version.parse("2.1.0")

```

Comment (volodymyrss) [reply README.md:130]: Or you mean version in own dependency? 
Well, in principle, if installed nominally, it should never happen that the version of the dependency deviates. I.e. there is no need for extra check or readme.

But if the problem justifies addition in the README, you might as well add an extra check in the code.

Comment (burnout87) [reply README.md:130]: > Or you mean version in own dependency? 

Yes, that is what I meant. To check if the installed version is different from the one listed in the `requirements`/`install_requires`

Comment (volodymyrss) [reply README.md:130]:
```
from pip._vendor import pkg_resources


_package_name = 'somepackage'
_package = pkg_resources.working_set.by_key[_package_name]

print([str(r) for r in _package.requires()])  # retrieve deps from setup.py
```

Comment (volodymyrss) [reply README.md:130]: Maybe add in readme that the dependency is in principle guaranteed with normal installation, and version is checked in runtime.

Comment (burnout87) [reply README.md:130]: Within the `__init__`, I now compare the two renku versions. So the check will be performed when importing the plugin

File: VERSION [hunk 2]
```diff
@@ -1 +1 @@
-0.0.1
+0.0.2
```
Comment (volodymyrss) [inline VERSION:1]: I think minor version should start from 1. I.e. the smallest initial version is `0.1.0`
(https://semver.org/)

File: README.md [hunk 3]
```diff
@@ -123,3 +123,22 @@ two json files have been loaded, and the related checkbox will enable/disable th
...
+
+```bash
+renku --version
+```
+
+Despite the dependency is guaranteed during the installation if the plugin, and additional check will be performed during the import.
```
Comment (volodymyrss) [inline README.md:136]: I am not sure this is a complete sentence. Despite => what? :)

Comment (burnout87) [reply README.md]: Yeah it made no sense...

**Review by volodymyrss — APPROVED** [review]

**Pull request merged**